analysis-pipe/04-Rstats02.R

```diff
@@ -345,10 +345,44 @@ if(!file.exists(paste0(indir,'/clustering.rdata')) | !file.exists(paste0(indir,'
 load(file = paste0(indir,"/scalingparams.rdata"))
 load(file=paste0(indir,'/centers.rdata'))
 load(file=paste(indir,'/clustering.rdata',sep=''))
-load(file=paste(indir,'/image_hold.rdata',sep=''))
```

Comment:
I do not understand this Hien'ism, why not:

load(file=paste(indir, 'image_hold.rdata', sep='/'))

?